### PR TITLE
Awesome books ES6 using Modules

### PR DESCRIPTION
**Pull request by lucas-crodrigues**
For this project, I was required to take last week's project and rewrite it using ES6 and modules.

The requirements were:

> ### Project requirements
> 
> - Setup your project:
>   - Create a new directory and initialize it with git.
>   - Use `npm init -y` command to create package.json file.
>   - Create the entry point for your JavaScript code called `index.js`
>   - Create `index.css` file for your styles.
>   - Create the main `index.html` file. Link your `index.js` and `index.css` files there.
>   - Create a directory called `modules`.
> - Re-create your _Awesome books_ app following the rules:
>   - Use ES6 syntax.
>   - Divide your code into modules. Save each module in a separate file in the `modules` dir. Import modules in the `index.js` file. For this exercise - the more modules the better!
>   - Keep all funcionalities of the app without errors.
>   - Refactor your methods and functions to arrow functions.
>   - Use let and const in a correct way.
> 
>  
> - Display the current date below the navigation bar. 
>   - For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
>   - Download the script and save it to your project. 
>   - Import it as a module.
>   - See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.

Thank you

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @lucas-crodrigues, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that has linter errors. You can just replace the linters with the javascript module llinters given. This will solve your problem.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by lucas-crodrigues** [comment]
> 

Sorry about that

File: modules/book-collection.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  static markupAllBooks() {
+    let allBooksHTML = '';
+    this.getLocalStorage().forEach((book, index) => {
+      allBooksHTML += `<div class="book" id="${book.title}">
+                        <p class="title">"${book.title}" by ${book.author} </p>
+                        <button type="button" class="shadow" id="r_${index}">Remove</button>
+                      </div>`;
+    });
+    return allBooksHTML;
+  }
```
Comment (thecodechaser) [inline modules/book-collection.js]: - [x] Please change this function to an arrow function to follow the ES6 syntax.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @lucas-crodrigues👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/book-collection.js (hunk 1 above)
Comment (lucas-crodrigues) [reply modules/book-collection.js]: Thank you, I missed this one

**Review by keroloussamy — APPROVED** [review]
Hi @lucas-crodrigues, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**